components/studio/bin/hab-studio.ps1

```diff
@@ -66,7 +67,8 @@ COMMON FLAGS:
     -n  Do not mount the source path into the Studio (default: mount the path)
     -q  Prints less output for better use in scripts
     -v  Prints more verbose output
-    -w  Use a Windows studio instead of a Docker Studio (only available on windows)
+    -D  Use a Docker Studio instead of a native Studio
+    -w  Use a Windows studio instead of a Docker Studio (DEPRECATED)
```

Comment:
Might be nice to indicate that it's just the _flag_ that's deprecated, and not the use of a Windows studio itself.